.github/workflows/terraform.yml

```diff
@@ -0,0 +1,39 @@
+name: 'Terraform'
+
+on:
+  pull_request:
+    branches:
+      - master
+
+env:
+  TF_VERSION: 0.14.3
```

Comment:
it's preferred to use something like this:
https://github.com/hmcts/cnp-azuredevops-libraries/blob/master/steps/terraform.yaml#L53-L57

that way the .terraform-version file can be automatically updated without this file having to be maintained